eng/scripts/Language-Settings.ps1

```diff
@@ -24,7 +24,12 @@ function Get-javascript-PackageInfoFromRepo ($pkgPath, $serviceDirectory)
     $jsStylePkgName = $projectJson.name.Replace("@", "").Replace("/", "-")
 
     $pkgProp = [PackageProps]::new($projectJson.name, $projectJson.version, $pkgPath, $serviceDirectory)
-    $pkgProp.SdkType = $projectJson.psobject.properties['sdk-type'].value
+    if ($projectJson.psobject.properties.name -contains 'sdk-type') {
```

Comment:
```suggestion
    if ($projectJson.psobject.properties -contains 'sdk-type') {
```

Reply:
This will not work as we need it to be the name of the properties.